core/unit_tests.c

```diff
@@ -60,6 +61,210 @@ unit_test_atomic_ops(void);
 void
 unit_test_jit_fragment_tree(void);
 
+#ifdef __AVX__
+
+void
+write_ymm_aux(dr_ymm_t *buffer, int regno)
+{
+    switch (regno) {
+#    define MOVE_TO_YMM_VEX(B, N)   \
```

Comment:
More readable if s/B/buf/;s/N/num/ or sthg instead of one-letter names.  Ditto below.

Reply:
Done.